activerecord/lib/active_record/test_databases.rb

```diff
@@ -8,31 +8,16 @@ module TestDatabases # :nodoc:
       create_and_load_schema(i, env_name: Rails.env)
     end
 
-    ActiveSupport::Testing::Parallelization.run_cleanup_hook do
-      drop(env_name: Rails.env)
-    end
-
     def self.create_and_load_schema(i, env_name:)
       old, ENV["VERBOSE"] = ENV["VERBOSE"], "false"
 
       ActiveRecord::Base.configurations.configs_for(env_name: env_name).each do |db_config|
         db_config.config["database"] += "-#{i}"
-        ActiveRecord::Tasks::DatabaseTasks.create(db_config.config)
```

Comment:
Where'd this go? Nothing in `reset_to_schema` sounds obviously likely to do the create if needed (and from its name, I don't think I'd expect it to anyway)

Reply:
I was thinking reset in the "`rake db:reset`" sense, which recreates the database, but reading it again I agree that might be confusing (reset implies migrations, we just want to load)

Reply:
I don't feel strongly about avoiding 'reset' -- that parallel makes sense too.

.. but where does the create actually happen now?

Reply:
Ack! You're right. This does not work when there is no DB 😳. I'll fix this and improve the naming.

Reply:
I went with `reconstruct_from_schema`, which I like (I didn't want recreate because create doesn't imply loading the schema, but recon<em>struct</em> does sound like it loads the DB's <em>struct</em>ure).

I've made sure this now also creates DBs.